rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -444,10 +404,12 @@ public String getTaskServerLogsByTag(String jobId, String tag)
     @Override
     public boolean isConnected() {
         boolean isConnected = false;
-        try {
-            isConnected = restApi().isConnected(sid);
-        } catch (NotConnectedRestException e) {
-            // ignore
+        if (initialized) {
+            try {
+                isConnected = restApi().isConnected(sid);
+            } catch (Throwable e) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1181)